cmd/minishift/cmd/openshift/set.go

```diff
@@ -37,8 +37,8 @@ const (
 	patchFlag  = "patch"
 
 	unknownPatchTargetError = "Unkown patch target. Only 'master' and 'node' are supported."
-	emptyPatchError         = "You need to specify a patch using --patch."
-	invalidJSONError        = "The specified patch need to be valid JSON."
+	emptyPatchError         = "You must specify a patch using the --patch flag."
+	invalidJSONError        = "You must specify a patch with a valid JSON format."
```

Comment:
" with a valid JSON format" sounds wrong to me. "Needs to be valid JSON". Also there is only one valid JSON format, not multiple which 'a' would imply.